freeipa/src/main/java/com/sequenceiq/freeipa/flow/stack/termination/StackTerminationFlowConfig.java

```diff
@@ -32,7 +34,8 @@
                     .defaultFailureEvent(TERMINATION_FAILED_EVENT)
                     .from(INIT_STATE).to(DEREGISTER_CLUSTERPROXY_STATE).event(TERMINATION_EVENT).noFailureEvent()
                     .from(DEREGISTER_CLUSTERPROXY_STATE).to(DEREGISTER_CCMKEY_STATE).event(CLUSTER_PROXY_DEREGISTRATION_FINISHED_EVENT).noFailureEvent()
-                    .from(DEREGISTER_CCMKEY_STATE).to(REMOVE_MACHINE_USER_STATE).event(CCM_KEY_DEREGISTRATION_FINISHED_EVENT).noFailureEvent()
+                    .from(DEREGISTER_CCMKEY_STATE).to(STOP_TELEMETRY_AGENT_STATE).event(CCM_KEY_DEREGISTRATION_FINISHED_EVENT).noFailureEvent()
+                    .from(STOP_TELEMETRY_AGENT_STATE).to(REMOVE_MACHINE_USER_STATE).event(STOP_TELEMETRY_AGENT_FINISHED_EVENT).noFailureEvent()
```

Comment:
this doesn't seem right in case of CCM. if we deregister freeipa from CCM and CP we won't be able to communicate with it anymore.